society-of-biblical-literature-fullnote-bibliography.csl

```diff
@@ -854,7 +882,18 @@
         <text macro="pages"/>
       </if>
       <else-if type="article-journal">
-        <text variable="locator" prefix=": "/>
+        <!-- Add support for individually paginated, electronic journal articles per https://sblhs2.com/2018/05/03/electronic-journals-with-individually-paginated-articles/ -->
+        <choose>
+          <if variable="number">
+            <text term="article-journal" form="verb-short" prefix=": "/>
+            <text variable="number" prefix=" " suffix=", "/>
+            <label variable="locator" form="short"/>
+            <text variable="locator"/>
```

Comment:
This should be more robust (the current version would look off for electronic articles without locators)

```suggestion
<group delimiter=" " prefix=": ">
            <text term="article-journal" form="verb-short"/>
            <text variable="number"/>
            <group prefix=", ">
            <label variable="locator" form="short"/>
            <text variable="locator"/>
            </group>
            </group>
```